src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -48,13 +49,17 @@
 public class AttachCaseCallbackService {
 
     public static final String INTERNAL_ERROR_MSG = "An error occurred";
+    public static final String PAYMENT_ERROR_MSG = "Payment reference can not be processed. Please try again later";
```

Comment:
This constant doesn't seem to be used anywhere.

Reply:
NO REVIEW for now

Reply:
> NO REVIEW for now

can we mark it somehow so reviewers are aware it's WIP